### ISSUE TITLE
Support `browsers` option

### ISSUE THREAD

**Issue by joeyhoer** [octopress/autoprefixer#8]
[Autoprefixer Rails](https://github.com/ai/autoprefixer-rails) supports a `browsers` option.

Would be great if we could add something like this is `_config.xml`:

```
autoprefixer:
  flexbox: no-2009
  browsers:
    - "> 1%"
    - "last 2 versions"
    - "IE > 8"
```

**Comment by digitalvapor** [comment]
seconded!

**Comment by nebulousdog** [comment]
would love to see this option

**Comment by imathis** [comment]
I just released 2.0.1; now `autoprefixer` from _config.yml will pass options to the processor. 🍻

**Comment by tom-spalding** [comment]
amazing! thanks @imathis
